group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupCoordinatorConfig.java

```diff
@@ -71,7 +71,7 @@ public class GroupCoordinatorConfig {
     /** New group coordinator configs */
     public static final String NEW_GROUP_COORDINATOR_ENABLE_CONFIG = "group.coordinator.new.enable";
     public static final String NEW_GROUP_COORDINATOR_ENABLE_DOC = "Enable the new group coordinator.";
-    public static final boolean NEW_GROUP_COORDINATOR_ENABLE_DEFAULT = false;
+    public static final boolean NEW_GROUP_COORDINATOR_ENABLE_DEFAULT = true;
```

Comment:
It is enabled by default now, so should we mark `group.coordinator.new.enable` as public config?

Reply:
We don't plan to let users choose the coordinator. Our plan is to make the new coordinator the default and only option starting from Apache Kafka 4.0. We will keep the internal config during for a few minor releases (as a safety net) but we will eventually remove it together with all the code of the old coordinator.

Reply:
> will eventually remove it together with all the code of the old coordinator.

happy to know it :)